.github/workflows/notify.yml

```
@@ -0,0 +1,10 @@
+name: Draco Insights Dashboard Notify
+on:
+  pull_request:
+  paths:
+    - 'locales/**/*'
+jobs:
+  notify:
```

You may want something like the following to avoid sending a Slack message for "manual" changes.

```suggestion
  notify:
    if: ${{ github.actor == 'brightspace-bot' }}
```

* https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif
* https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

---

I'll include this as a second PR to make sure it sends the push first, then add the bot condition